# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1285390812 25200
#      Fri Sep 24 22:00:12 2010 -0700
# Node ID 630dc1933fe60284a54a423b51e150cc5c3f7d17
# Parent  43c1f914e293fa04ce8df3cdeabb3655a06979f8
Rephrase output from 'make check' for clarity.

diff --git a/test/ChangeLog b/test/ChangeLog
--- a/test/ChangeLog
+++ b/test/ChangeLog
@@ -1,8 +1,12 @@
+2010-09-24  Rik <octave@nomad.inbox5.com>
+
+	* fntests.m: Rephrase output for clarity.
+
 2010-08-17  Rik <octave@nomad.inbox5.com>
 
 	* fntests.m: Add extra newline for more readable output.
 
 2010-07-30  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: New clean-local target to remove directories 
 	created automatically during testing.
diff --git a/test/fntests.m b/test/fntests.m
--- a/test/fntests.m
+++ b/test/fntests.m
@@ -251,18 +251,18 @@ try
     printf ("\nThere %s %d expected %s (see fntests.log for details).\n",
 	    t1, dxf, t2);
     puts ("\nExpected failures are known bugs.  Please help improve\n");
     puts ("Octave by contributing fixes for them.\n");
   endif
   if (dsk > 0)
     printf ("\nThere were %d skipped tests (see fntests.log for details).\n", dsk);
     puts ("Skipped tests are features that are disabled in this version\n");
-    puts ("of Octave as the needed libraries were not present when Octave\n");
-    puts ("was built\n");
+    puts ("of Octave because the needed libraries were not present when Octave\n");
+    puts ("was built.\n");
   endif
 
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".m");
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".cc");
 
   puts ("\nPlease help improve Octave by contributing tests for\n");
   puts ("these files (see the list in the file fntests.log).\n\n");
 
